pkgs/by-name/om/omnix/package.nix

```diff
@@ -0,0 +1,132 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  installShellFiles,
+  openssl,
+  cachix,
+  versionCheckHook,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "omnix";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "juspay";
+    repo = "omnix";
+    tag = finalAttrs.version;
+    hash = "sha256-vpjYVfs3gS+gkMSgeR3BH4hTLmi/fkWHmCKvSjFq7cg=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-Yv5X/lZdQ2vdxLzSPeLx5TNapKtCIQNc6Wy0I+4q4CI=";
+
+  nativeBuildInputs = [
+    installShellFiles
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  # Note: The ENVs below will have to be kept in sync with <https://github.com/juspay/omnix/blob/main/nix/envs/default.nix>
+  OMNIX_SOURCE = finalAttrs.src;
```

Comment:
Shouldn't we use the attribute `env` for all these env. variables ?

Reply:
Done